
Well - they already commented "Only include this when necessary. What is this
really dependent on ?". I think the answer is LESSTIF_PRODUCTION. Since
we enable "production" in configure we need to not include this function
since the ...DebugFile is not defined ...

  - Rene Rebe <rene@rocklinux.org>

--- lesstif-0.93.96/lib/Xm-2.1/DebugUtil.c.orig	2004-09-29 11:17:26.904106539 +0000
+++ lesstif-0.93.96/lib/Xm-2.1/DebugUtil.c	2004-09-29 11:17:51.370195643 +0000
@@ -2517,6 +2517,7 @@
  * Only include this when necessary.
  * What is this really dependent on ?
  */
+#ifndef LESSTIF_PRODUCTION
 extern void
 _LtDebugPrintStackTrace()
 {
@@ -2534,3 +2535,4 @@
 	fflush(_LtDebugFile);
 #endif
 }
+#endif
